builtin/game/chatcommands.lua

```diff
@@ -530,11 +530,16 @@ core.register_chatcommand("giveme", {
 })
 
 core.register_chatcommand("spawnentity", {
-	params = "<EntityName>",
-	description = "Spawn entity at your position",
+	params = "<EntityName> [<X>,<Y>,<Z>]",
+	description = "Spawn entity at given (or your) position",
 	privs = {give=true, interact=true},
 	func = function(name, param)
-		local entityname = string.match(param, "(.+)$")
+		local entityname = string.match(param, "^([^ ]+)$")
+		local p
+		if not entityname then
```

Comment:
does nobody see this? there is two `if not entityname then`,
what in the world is it trying to match above anyway? a space?,
what's wrong with the old regex?
i think it should use split(" ") like `deleteblocks` instead to separate the entityname and pos